src/Components/ManageTeam/ConfirmDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle } from '@mui/material';
```

Comment:
I'd suggest using lower case for the name of the folder `Components` to make it consistent with the naming convention of the rest of the folders in the project.